src/serena/agent.py

```diff
@@ -536,6 +536,49 @@ def apply(
         return self._limit_length(result, max_answer_chars)
 
 
+class GetReferencingCodeExtractsTool(Tool):
```

Comment:
I think I would prefer the term "excerpt" or even "snippet" over "extract".
Also, I think we should prefer the verb "find" over "get".

Reply:
Done